packages/tailwindcss/src/index.test.ts

```diff
@@ -1120,9 +1120,135 @@ describe('Parsing themes values from CSS', () => {
         ['bg-tomato', 'bg-potato', 'bg-avocado'],
       ),
     ).toThrowErrorMatchingInlineSnapshot(
-      `[Error: Files imported with \`@import "…" reference\` must only contain \`@theme\` blocks.]`,
+      `[Error: Files imported with \`@import "…" theme(…)\` must only contain \`@theme\` blocks.]`,
     )
   })
+
+  test('theme values added as `inline` are not wrapped in `var(…)` when used as utility values', () => {
+    expect(
+      compileCss(
+        css`
+          @theme inline {
+            --color-tomato: #e10c04;
+            --color-potato: #ac855b;
+            --color-primary: var(--primary);
+          }
+
+          @tailwind utilities;
+        `,
+        ['bg-tomato', 'bg-potato', 'bg-primary'],
+      ),
+    ).toMatchInlineSnapshot(`
+      ":root {
+        --color-tomato: #e10c04;
+        --color-potato: #ac855b;
+        --color-primary: var(--primary);
+      }
+
+      .bg-potato {
+        background-color: #ac855b;
+      }
+
+      .bg-primary {
+        background-color: var(--primary);
+      }
+
+      .bg-tomato {
+        background-color: #e10c04;
+      }"
+    `)
+  })
+
+  test('wrapping `@theme` with `@media theme(inline)` behaves like `@theme inline` to support `@import` statements', () => {
+    expect(
+      compileCss(
+        css`
+          @media theme(inline) {
+            @theme {
+              --color-tomato: #e10c04;
+              --color-potato: #ac855b;
+              --color-primary: var(--primary);
+            }
+          }
+
+          @tailwind utilities;
+        `,
+        ['bg-tomato', 'bg-potato', 'bg-primary'],
+      ),
+    ).toMatchInlineSnapshot(`
+      ".bg-potato {
```

Comment:
I think this should contain the `:root` with the variables. It currently behaves as if we wrote `@media theme(inline reference)`.

```css
:root {
  --color-tomato: #e10c04;
  --color-potato: #ac855b;
  --color-primary: var(--primary);
}
```

Reply:
Good catch, fixed! (By you, really 😄)